### DISCUSSION TITLE
Error: No targets found. You may need to wait longer, or you may have issues with your wifi card (ath9k driver)

### DISCUSSION THREAD

**Discussion by carmichaelcontracting** [aircrack-ng/aircrack-ng#2564] [Q&A]
hello
I run Debian linux, with a unpatched wifi network card,  That is a Qualcomm Atheros
AR9565 Wireless Network Adapter (rev 01)ath9k		Qualcomm Atheros QCA9565

it seems that I cannot use aircrack-ng without a patched driver

Where can I get such a driver?
how can I install such a patched driver?

thank you

**Comment by Mister-X-** [comment]
What issue are you experiencing that makes you say that you cannot use aircrack-ng without a patched driver?
